jenkins/freestyle/get-sitespeed-report.sh

```diff
@@ -35,84 +35,88 @@ set -e
 : ${CONNECTION:?"The CONNECTION environment variable must be set."}
 : ${SITESPEED_BROWSER:?"The SITESPEED_BROWSER environment variable must be set."}
 
-if [ $GET_URLS_FROM_FILE == "true" ] ; then
-	msg="The SITESPEED_URL_FILE environment variable must be set to the file containing the URLs to test."
-	: ${SITESPEED_URL_FILE:?$msg}
+# Test for the existence of TEST_URL_FILE
+if [ ! -z ${TEST_URL_FILE+x} ] ; then
 
-	if [ ! -f $SITESPEED_URL_FILE ]; then
-        echo "ERROR: Specified SITESPEED_URL_FILE '$SITESPEED_URL_FILE' does not exist."
-        exit 1
-	fi
+    if [ "$TEST_URL_FILE" == "" ] ; then
+        echo "TEST_URL_FILE environment variable is set, but empty. Skipping related logic."
 
-	# HACK: Copy the contents of the URL file to a file
-	# named "multiple_urls" and use that so that the output folder will
-	# have a known name ("multiple_urls") and thus the artifacts can be
-	# archived correctly on jenkins
-	TMP_URL_FILE="/tmp/multiple_urls"
-	cp -rf $SITESPEED_URL_FILE $TMP_URL_FILE
+    else
 
-	if [ ! -f $TMP_URL_FILE ]; then
-        echo "ERROR: Could not copy '$SITESPEED_URL_FILE' to '$TMP_URL_FILE'."
-        exit 1
-	fi
+        # HACK: Copy the contents of the URL file to a file
+        # named "multiple_urls" and use that so that the output folder will
+        # have a known name ("results") and thus the artifacts can be
```

Comment:
the known name would be "multiple_urls". Feel free to change up the wording, I know it was a bit awkward.